### ISSUE TITLE
Set up git hooks for testing and linting

### ISSUE THREAD

**Issue by yourbuddyconner** [CodaProtocol/coda-automation#35]
We should be testing our changes to code in a sane and programmatic way. At the very least it'd be nice to set up automatic formatting via these git hooks: https://github.com/antonbabenko/pre-commit-terraform
